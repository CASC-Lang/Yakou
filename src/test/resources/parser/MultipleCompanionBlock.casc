class MultipleCompanionBlock {

}

impl MultipleCompanionBlock {
    comp {

    }

    comp {

    }
}