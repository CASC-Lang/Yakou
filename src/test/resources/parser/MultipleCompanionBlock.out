error: Duplicate `comp` block
--> ./MultipleCompanionBlock.casc:10:4
9  |
10 |     comp {
         ^^^^= hint: Remove this block
11 |
